SPECS/nghttp2/nghttp2.spec

```diff
@@ -58,6 +62,9 @@ rm %{buildroot}/%{_libdir}/*.la
 %{_libdir}/pkgconfig/*.pc
 
 %changelog
+* Tue Nov 03 2020 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.41.0-2
```

Comment:
Can we update this date?

Reply:
Good Catch .. did not even notice the date was incorrect :(

Reply:
```suggestion
* Wed Jun 23 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.41.0-2
```

Reply:
Thanks, guys!